build.assets/tooling/cmd/difftest/ast.go

```diff
@@ -0,0 +1,199 @@
+package main
```

Comment:
Can you add our license header to all .go source files? You can grab it from one of .go files in the code.